modules/optimizing-mtu-networking.adoc

```diff
@@ -7,6 +7,6 @@
 
 There are two important maximum transmission units (MTUs): the network interface controller (NIC) MTU and the cluster network MTU.
 
-The NIC MTU is only configured at the time of {product-title} installation. The MTU must be less than or equal to the maximum supported value of the NIC of your network. If you are optimizing for throughput, choose the largest possible value. If you are optimizing for lowest latency, choose a lower value.
+The NIC MTU is configured at the time of {product-title} installation and you can also change the xref:../../networking/changing-cluster-network-mtu.adoc#changing-cluster-network-mtu[cluster's MTU] as a Day 2 operation. The MTU must be less than or equal to the maximum supported value of the NIC of your network. If you are optimizing for throughput, choose the largest possible value. If you are optimizing for lowest latency, choose a lower value.
```

Comment:
Also, we can't use xrefs in a module, so if you want to like to this page it would have to be in an additional resources section under this module

Reply:
Whoops